microsoft-edge/webview2/concepts/security.md

```diff
@@ -0,0 +1,33 @@
+---
+description: Understand how to develop secure WebView2 applications
+title: Best practices for developing secure WebView2 applications
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/21/2020
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html, security
+---
+
+# Best practices for developing secure WebView2 applications
+
+When used correctly, hosting web content inside your native application offers certain advantages, such as ___. To avoid vulnerabilities that can arise from hosting web content, make sure to design your WebView2 application to closely monitor interactions between the web content and the host application. 
```

Comment:
Can you replace ___ with text please?

Reply:
There are more than just a few really depends on the use case want to avoid listing things out, doing so would only narrow the amount of use cases

Reply:
LGTM!